### PR TITLE
Feat: Bid list for admin.

### PR DESCRIPTION
**Pull request by subbu3012**

### CODE REVIEW

File: src/app/pages/buyer/buyer-bid-list/buyer-bid-list.component.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export class BuyerBidListComponent implements OnInit, OnDestroy {
    */
   private initializeBidList(buyerId: number) {
     this.appServ.getBidsForBuyer(buyerId).subscribe(data => {
-      this.bids = data;
+      this.bids = data.sort((a, b) => b.id - a.id)
```
Comment (vraravam) [inline src/app/pages/buyer/buyer-bid-list/buyer-bid-list.component.ts]: @subbu3012 - sorting of data with paginated results won't work. Shouldn't this sort be applied at the backend?

File: src/app/pages/seller/seller-bid-list/seller-bid-list.component.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export class SellerBidListComponent implements OnInit {
     this.seller = this.appServ.allSellers.filter(seller => seller.id == this.sellerId)[0];
 
     this.appServ.getAllBids().subscribe(data => {
-      this.bids = data.filter(bid => bid.sellerId == +this.sellerId);
+      this.bids = data.filter(bid => bid.sellerId == +this.sellerId).sort((a, b) => b.id - a.id)
```
Comment (vraravam) [inline src/app/pages/seller/seller-bid-list/seller-bid-list.component.ts]: @subbu3012 - same as the previous: sorting in the frontend for paginated data from the backend will not work.

File: src/app/pages/buyer/buyer-bid-list/buyer-bid-list.component.ts (hunk 1 above)
Comment (subbu3012) [reply src/app/pages/buyer/buyer-bid-list/buyer-bid-list.component.ts]: @vraravam to be frank, both sorting and pagination should be done on backend. doing this on frontend has two reasons:
1. we havent done pagination on backend (yet). 
2. azhagu wanted this to be done very badly as there is a drastic incoming of bids after launch. 

coming back to how this code works, the sorting is done on the complete data (the apis response) and then, that sorted data is paginated. the reverse (paginating and then sorting) wont work as you said.

File: src/app/pages/seller/seller-bid-list/seller-bid-list.component.ts (hunk 2 above)
Comment (subbu3012) [reply src/app/pages/seller/seller-bid-list/seller-bid-list.component.ts]: @vraravam please refer to the above comment. https://github.com/chennaitricolor/wastexchange-fe/pull/41#discussion_r358742292

Comment (vraravam) [reply src/app/pages/seller/seller-bid-list/seller-bid-list.component.ts]: Got it

File: src/app/pages/buyer/buyer-bid-list/buyer-bid-list.component.ts (hunk 1 above)
Comment (vraravam) [reply src/app/pages/buyer/buyer-bid-list/buyer-bid-list.component.ts]: Yes-  now that I see the code - the backend doesn't do both these functions. We should play cards to accomplish both these CFRs in the backend. Approving with that being considered as "tech debt"

**Pull request merged**